test-util/mock-http.ts

```diff
@@ -36,7 +41,9 @@ const mockEndpoint: EndpointOptions = {
 };
 
 export function mockAiCoreEnvVariable(): void {
```

Comment:
[info] unrelated to this PR, but I think this method is not required anymore, since its mocked in the global setup

Reply:
you're right, it wasn't used anywhere anyways